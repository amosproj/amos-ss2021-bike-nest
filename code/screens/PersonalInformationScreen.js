import EditPersonalInformation from "../components/EditPersonalInformation/EditPersonalInformation";
import EditPassword from "../components/EditPersonalInformation/EditPassword";
import React, {Component} from "react";
import {Text, View} from "react-native";

export default class PersonalInformationScreen extends Component{
    render(){
        return (
            <View>
                <h1>Personal settings</h1>
                <EditPersonalInformation></EditPersonalInformation>
                <EditPassword></EditPassword>
            </View>
        )
    }
}
