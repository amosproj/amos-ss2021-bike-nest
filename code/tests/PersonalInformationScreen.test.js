import React from 'react';
import renderer from 'react-test-renderer';
import PersonalInformationScreen from '../screens/PersonalInformationScreen';

it('renders correctly', () => {
    const tree = renderer
        .create(<PersonalInformationScreen />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});