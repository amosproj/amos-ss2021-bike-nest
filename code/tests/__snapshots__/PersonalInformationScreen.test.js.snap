// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <RCTScrollView>
    <View>
      <Text
        style={
          Array [
            Object {
              "fontSize": 28,
              "fontWeight": "bold",
              "margin": 30,
              "padding": 15,
            },
            Object {
              "alignSelf": "center",
              "justifyContent": "center",
            },
          ]
        }
      >
        Einstellungen
      </Text>
      <Image
        source={1}
        style={
          Object {
            "alignSelf": "center",
            "height": 120,
            "resizeMode": "contain",
            "width": 120,
          }
        }
      />
      <View
        style={
          Object {
            "backgroundColor": "#ffffff",
            "borderRadius": 15,
            "marginTop": 10,
          }
        }
      />
      <View
        style={
          Object {
            "backgroundColor": "#fffff",
            "borderRadius": 15,
            "marginTop": 25,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "flex": 1,
            }
          }
        >
          <Modal
            animationType="slide"
            hardwareAccelerated={false}
            onRequestClose={[Function]}
            transparent={true}
            visible={false}
          >
            <View
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "#00000090",
                  "flex": 1,
                  "flexDirection": "column",
                  "justifyContent": "center",
                }
              }
            >
              <View
                style={
                  Object {
                    "alignItems": "center",
                    "backgroundColor": "#fff",
                    "borderRadius": 15,
                    "elevation": 3,
                    "height": 184,
                    "justifyContent": "space-between",
                    "padding": 18,
                    "width": 288,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "fontSize": 18,
                      "fontWeight": "300",
                    }
                  }
                >
                  
                </Text>
                <Text
                  style={
                    Object {
                      "fontSize": 14,
                    }
                  }
                >
                  
                </Text>
                <View
                  accessible={true}
                  focusable={true}
                  onClick={[Function]}
                  onResponderGrant={[Function]}
                  onResponderMove={[Function]}
                  onResponderRelease={[Function]}
                  onResponderTerminate={[Function]}
                  onResponderTerminationRequest={[Function]}
                  onStartShouldSetResponder={[Function]}
                  style={
                    Object {
                      "alignItems": "center",
                      "backgroundColor": "#F28D35",
                      "borderRadius": 38,
                      "elevation": 3,
                      "height": 55,
                      "justifyContent": "center",
                      "margin": 9,
                      "opacity": 1,
                      "padding": 18,
                      "width": 128,
                    }
                  }
                >
                  <View
                    style={
                      Object {
                        "alignItems": "center",
                        "flexDirection": "row",
                      }
                    }
                  >
                    <Text
                      style={
                        Object {
                          "color": "#F2F2F2",
                          "fontSize": 14,
                          "height": "100%",
                          "textAlign": "center",
                          "width": "100%",
                        }
                      }
                    >
                      Ok
                    </Text>
                  </View>
                </View>
              </View>
            </View>
          </Modal>
          <Text />
          <View>
            <Text
              style={
                Object {
                  "color": "#333333",
                  "justifyContent": "center",
                  "margin": 5,
                  "paddingLeft": 12,
                  "textAlignVertical": "center",
                  "width": 328,
                }
              }
            >
              Password:
            </Text>
          </View>
          <View>
            <Text
              style={
                Object {
                  "color": "#333333",
                  "justifyContent": "center",
                  "margin": 5,
                  "paddingLeft": 12,
                  "textAlignVertical": "center",
                  "width": 328,
                }
              }
            >
              *****
            </Text>
          </View>
          <View
            accessible={true}
            focusable={true}
            onBlur={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "#ffffff",
                "borderRadius": 38,
                "elevation": 3,
                "height": 55,
                "justifyContent": "center",
                "margin": 9,
                "width": 328,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": undefined,
                  "fontSize": 14,
                }
              }
            >
              Edit
            </Text>
          </View>
        </View>
      </View>
    </View>
  </RCTScrollView>
  <View
    style={
      Object {
        "flexDirection": "row",
        "height": 70,
        "justifyContent": "flex-start",
        "width": "100%",
      }
    }
  >
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#F28D35",
          "flex": 1,
          "opacity": 1,
          "padding": 5,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "flexDirection": "column",
            "justifyContent": "flex-start",
          }
        }
      >
        <View
          style={
            Object {
              "margin": 5,
            }
          }
        >
          <Image
            source={1}
          />
        </View>
        <Text
          style={
            Object {
              "color": "#F2F2F2",
              "fontSize": 14,
              "height": "100%",
              "textAlign": "center",
              "width": "100%",
            }
          }
        >
          Profil
        </Text>
      </View>
    </View>
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#F28D35",
          "flex": 1,
          "opacity": 1,
          "padding": 5,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "flexDirection": "column",
            "justifyContent": "flex-start",
          }
        }
      >
        <View
          style={
            Object {
              "margin": 5,
            }
          }
        >
          <Image
            source={1}
          />
        </View>
        <Text
          style={
            Object {
              "color": "#F2F2F2",
              "fontSize": 14,
              "height": "100%",
              "textAlign": "center",
              "width": "100%",
            }
          }
        >
          (un)lock
        </Text>
      </View>
    </View>
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#F28D35",
          "flex": 1,
          "opacity": 1,
          "padding": 5,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "flexDirection": "column",
            "justifyContent": "flex-start",
          }
        }
      >
        <View
          style={
            Object {
              "margin": 5,
            }
          }
        >
          <Image
            source={1}
          />
        </View>
        <Text
          style={
            Object {
              "color": "#F2F2F2",
              "fontSize": 14,
              "height": "100%",
              "textAlign": "center",
              "width": "100%",
            }
          }
        >
          Navigation
        </Text>
      </View>
    </View>
    <View
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "backgroundColor": "#F28D35",
          "flex": 1,
          "opacity": 1,
          "padding": 5,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "flexDirection": "column",
            "justifyContent": "flex-start",
          }
        }
      >
        <View
          style={
            Object {
              "margin": 5,
            }
          }
        >
          <Image
            source={1}
          />
        </View>
        <Text
          style={
            Object {
              "color": "#F2F2F2",
              "fontSize": 14,
              "height": "100%",
              "textAlign": "center",
              "width": "100%",
            }
          }
        >
          Verlauf
        </Text>
      </View>
    </View>
  </View>
</View>
`;
